src/System.Private.CoreLib/src/Internal/Resources/WindowsRuntimeResourceManagerBase.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+
+namespace Internal.Resources
+{
+    //
+    // This is implemented in System.Runtime.WindowsRuntime as function System.Resources.WindowsRuntimeResourceManager,
+    // allowing us to ask for a WinRT-specific ResourceManager.
+    // It is important to have WindowsRuntimeResourceManagerBase as regular class with virtual methods and default implementations. 
```

Comment:
This comment does not match the implementation. The comment stays that it is important to not have the class abstract; but the implementation is actually abstract.

Reply:
I think it is fine to keep them abstract since we plan to always ship them together.

Reply:
Remove these confuse comments.